rmf_site_editor/src/site/fuel_cache.rs

```diff
@@ -71,17 +87,27 @@ pub fn handle_update_fuel_cache_requests(
                 #[cfg(not(target_arch = "wasm32"))]
                 let write_to_disk = true;
                 // Send client if update was successful
-                let res = fuel_client.update_cache(write_to_disk).await;
+
+                let res = fuel_client
+                    .update_cache_with_progress(write_to_disk, Some(progress))
+                    .await;
                 sender
                     .send(FuelCacheUpdated(res))
                     .expect("Failed sending fuel cache update event");
             })
             .detach();
     }
+
+    while let Ok(next_model) = progress_channel.receiver.try_recv() {
+        info!(
+            "Received model {} owned by {}",
+            next_model.name, next_model.owner,
+        );
+    }
```

Comment:
Noted on the message being misleading. I'm not sure what would be the most accurate thing to say, but I tweaked it here: [2fe3acd](https://github.com/open-rmf/rmf_site/pull/229/commits/2fe3acddc038f14e0b229f3f39e90f35aa477acb)

I definitely recognize that it's spammy, but I'm pretty okay with it as a first draft. The log will always show the latest messages first and foremost, so the long list of messages won't wash out any new messages that are coming in. I feel that the current blast of messages is at least an improvement over the eerie silence that used to be there.

I would suggest we open a ticket on this to revisit after this PR instead of blocking on this.